meshroom/nodes/aliceVision/CameraInit.py

```diff
@@ -204,6 +211,13 @@ class CameraInit(desc.CommandLineNode):
             value=True,
             uid=[0],
         ),
+        desc.BoolParam(
+            name='noColorProfileIsError',
```

Comment:
Avoid negation in all variables name.
Maybe something like "errorOnMissingColorProfile"?